packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1413,6 +1413,155 @@ The following conditions may be applied to the Rules associated with Password Po
 
 * [Network Condition](#network-condition-object)
 
+
+### Updates to Password Policy For Identity Engine
+
+<ApiLifecycle access="ie" /><br>
+> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+
+This updated Password Policy API shows only the changes from the original [Password Policy API](#password-policy) above.
+Earlier, we supported setting Recovery Factors on the [Policy](#policy-settings-example-2). With Identity Engine, the new Password Policy model does not contain Recovery Factors but instead Password Policy Rule model contains them.
```

Comment:
This seems like it should require the change of one or more objects that are documented?